docs/refarch/boundedContexts/fspInteropApi/index.md

```diff
@@ -16,9 +22,13 @@ Terms with specific and commonly accepted meaning within the Bounded Context in
 
 | Term        | Description                                                                                                                                                      |
 | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Participant | Financial Service Provider (FSP) that has registered on the Mojaloop ecosystem. Allowing said FSP to be able to transact with other Participants                             |
+| (D)FSP      | (Digital) Financial Service Provider                                                                                                                             |
+| Participant | Financial Service Provider (FSP) that has registered on the Mojaloop ecosystem. Allowing said FSP to be able to transact with other Participants                 |
 | FSP IOP API | Financial Service Provider Interoperability Application Programming Interface, the API that provides access to functions and features of the Mojaloop Ecosystem. |
-| (D)FSP        | (Digital) Financial Service Provider                                                                                                                               |
+| Payer       | The payer of electronic funds in a payment transaction.                                                                                                          |
+| Payer FSP   | Payer's Financial Service Provider.                                                                                                                              |
+| Payee       | The recipient of electronic funds in a payment transaction.                                                                                                      |
```

Comment:
I assume this is the "Payee FSP's Customer"?

Reply:
Yes that would be the party, definition based on:
https://docs.mojaloop.io/mojaloop-specification/fspiop-api/documents/Glossary.html#3-api-glossary